src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -314,7 +314,7 @@ Just set the `pinned` prop.
 				<!-- Counter and Actions -->
 				<div v-if="hasUtils && !editingActive"
```

Comment:
would it also be possible to not render the element at all instead of hiding visually?

Reply:
The CSS solution uses information on the hover state that vue doesn't have access to, so this is probably the most pragmatic way.